Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM condaforge/mambaforge
+
+RUN conda update -n base conda
+RUN conda install -n base conda-libmamba-solver
+RUN conda config --set solver libmamba
+
+RUN apt-get update && apt-get install -y bash git && apt-get install gcc -y
+
+WORKDIR /pypsa-earth
+
+COPY ./envs ./temp
+
+RUN conda env create -n pypsa-earth -f temp/linux-pinned.yaml
+
+RUN conda init bash
```

Comment:
@davide-f  and @ekatef ,
We are currently using the linux-pinned.yaml file to create the environment. Will these issues be resolved in future once we update the Linux-pinned configuration? 
or should we try other way

Reply:
You mean the recent environment issues? A solution has still been found, we only need to align on implementing it (realistically speaking, it should be done by tomorrow European evening)